src/components/AddRestaurantModal/AddRestaurantModal.jsx

```diff
@@ -1,15 +1,43 @@
+import { useState } from "react";
 import "./AddRestaurantModal.css";
 
-function AddRestaurantModal() {
+function AddRestaurantModal({ onClose, onAdd }) {
+  const [name, setName] = useState("");
+  const [description, setDescription] = useState("");
+  const [category, setCategory] = useState("");
+
+  const handleAdd = (e) => {
```

Comment:
```suggestion
  // 음식점 추가 폼을 제출하면 이 함수가 동작할 것이다.
  const handleRestaurantAddFormSubmit = (e) => {
```
저는 개인적으로 함수명이 길어지더라도 함수명으로 동작을 명확하게 설명해주는게 좋다고 생각합니다🙃🙃🙃

지금 함수명은 많이 추상화되어 있어서 어떤 동작을 하는지 나만 아는 느낌? 동료 개발자가 봤을 때는 어떤 동작을 하는지 한 눈에 알아보기 어려울 것 같아요🤔

Reply:
그러네용 변수명을 너무 추상적으로 지었다는 느낌이 들어요!!! 지적해주셔서 감사합니다 ㅎㅎ